WcaOnRails/lib/database_dumper.rb

```diff
@@ -745,7 +745,10 @@ def self.development_dump(dump_filename)
           column_sanitizer == :copy ? "#{table_name}.#{column_name}" : "#{column_sanitizer} as #{column_name}"
```

Comment:
Does this `as #{column_name}` part not need escaping?

Reply:
Hmm, you mean the sanitized part? I originally skipped that as "rank" is unlikely to be sanitized, but you're right, if we're going it as a general solution we might as well future-proof it.